content/docs/started/k8s/kfctl-existing-arrikto.md

```diff
@@ -420,6 +554,81 @@ This section focuses on setting up Dex to authenticate with an existing LDAP dat
           * Add or update a label on the PodTemplate.
           * Save the deployment to trigger a rolling update.
 
+### Troubleshooting
+
+If the Kubeflow Dashboard is not available at `https://<LoadBalancerIP>` ensure that:
```

Comment:
s/Dashboard/dashboard

(Because capitalising things can be confusing - it makes readers think they should see the brand name "Kubeflow Dashboard" somewhere on the UI.")

I see we call it a "Dashboard" earlier on this page too. Please would you change any existing occurrences while you're here?

Reply:
I agree and changed two more occurrences